_component_design/chips.md

```diff
@@ -17,27 +17,63 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## When to use
 
-### When to use
+Use a chips to represent graphically a block of information. For example, use chips to represent a user or device.
 
-Chips are best used with indicating a list of items that may be part of a different workflow. For example, adding or removing a list of requestors on a ticket.
+## Best practices
 
-### Best practices
+Consider the following best practices when constructing a chip:
 
 - Avoid using chips in multiple locations on a page.
-- Ensure the user can concretely identify the item represented by the chip.
+- Label the chip so that the user can identify the item represented by the chip.
```

Comment:
Is the chip representing an item, a unit, a block of information? You could replace "...item..." with "...information...".